.changeset/curly-plants-grab.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": patch
+---
+
+Adapt styling of `ContentScopeIndicator` on mobile to align with the Comet DXP design
```

Comment:
I think in this case it makes sense to explicitly mention the issue that is being fixed